src/site/content/en/blog/mouse-acceleration/index.md

```diff
@@ -0,0 +1,175 @@
+tell you how much the mouse pointer moved since the last move event. However,
+those are not updated when the pointer moves outside of the web page.
+
+```js
+document.addEventListener("mousemove", (event) => {
+  console.log(`movementX: ${event.movementX} movementY: ${event.movementY}`);
+});
+```
+
+Capturing the mouse pointer (or requesting a pointer lock) allows you to not
+worry about the pointer moving outside anymore. This is especially useful for
+immersive web games. When the pointer is locked, all mouse events go to the
+target element of the pointer lock.
+
+Call `requestPointerLock()` on the target element to request a pointer lock, and
+listen to `pointerlockchange` and `pointerlockerror` events to monitor pointer
+lock changes.
+
+```js
+const myTargetElement = document.body;
+
+// Call this function to request a pointer lock.
+function requestPointerLock() {
+  myTargetElement.requestPointerLock();
+}
+
+document.addEventListener("pointerlockchange", () => {
+  if (document.pointerLockElement) {
+    console.log(`pointer is locked on ${document.pointerLockElement}`);
+  } else {
+    console.log("pointer is unlocked");
+  }
+});
+
+document.addEventListener("pointerlockerror", () => {
+  console.log("pointer lock error");
+});
+```
+
+### Disable mouse acceleration {: #disable-mouse-acceleration }
+
+Call `requestPointerLock()` with  `{ unadjustedMovement: true }` to disable
+OS-level adjustment for mouse acceleration, and access raw mouse input.
+This way, mouse movement data from `mousemove` events won't include mouse
+acceleration when the pointer is locked.
+
+Use the new returned promise from `requestPointerLock()` to know if the request
+was successful.
+
+```js
+function requestPointerLockWithUnadjustedMovement() {
+  const promise = myTargetElement.requestPointerLock({
+    unadjustedMovement: true,
+  });
+
+  if (!promise) {
+    console.log("disabling mouse acceleration is not supported");
+    return;
+  }
+
+  return promise
+    .then(() => console.log("pointer is locked"))
+    .catch((error) => {
+      if (error.name === "NotSupportedError") {
+        // Some platforms may not support unadjusted movement.
+        // You can request again a regular pointer lock.
+        return myTargetElement.requestPointerLock();
+      }
+    });
+}
+```
+
+It is possible to toggle between accelerated and non-accelerated mouse movement
+data without releasing the pointer lock. Simply request the pointer lock again
+with the desired option. If that request fails, the original lock will remain
+intact and the returned promise will reject. No pointer lock events will fire
+for a failed change request.
+
+### Browser support {: #browser-support }
+
+The Pointer Lock API is [well supported across browsers]. However Chromium-based
+browsers (e.g. Chrome, Edge, etc.) are the only ones to support disabling
+OS-level adjustment for mouse acceleration at the time of writing.
```

Comment:
```suggestion
OS-level adjustment for mouse acceleration as of October 2020.
See MDN's [Browser compatibility](https://developer.mozilla.org/en-US/docs/Web/API/Pointer_Lock_API#Browser_compatibility)
table for updates.
```
@beaufortfrancois can you create an issue on BCD asking them to add `unadjustedMovement`? https://github.com/mdn/browser-compat-data/issues